### ISSUE TITLE
AbstractNativeClientBuilder: Anonymous new DnsResolver() can be replaced with lambda

### ISSUE THREAD

**Issue by manusa** [eclipse-jkube/jkube#3369]
### Description

The class [AbstractNativeClientBuilder](https://github.com/eclipse-jkube/jkube/blob/5da6068d7756784aef9569568ee80f98da97e296/jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/access/hc/util/AbstractNativeClientBuilder.java#L81-L86) presents the issue: Anonymous new DnsResolver() can be replaced with lambda.

The following line:
https://github.com/eclipse-jkube/jkube/blob/5da6068d7756784aef9569568ee80f98da97e296/jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/access/hc/util/AbstractNativeClientBuilder.java#L81-L86
should be changed to:

```java
        return host -> new InetAddress[] {null};
```
### :red_circle: Before you start :red_circle:

:point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down:

Make sure you read the [contributing guide](https://www.eclipse.org/jkube/contributing) first.

Pay special attention to the ECA agreement section and the requirement to sign-off your commit.

This is a [first-timers-only](https://eclipse.dev/jkube/contributing/#first-timers-only-issues) issue. These issues are especially created for contributors who are contributing to the project for the **first time**.

If you have already contributed to this project in the past, please don't ask to be assigned one of these issues and look for [good-first-issues](https://github.com/eclipse-jkube/jkube/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) or [help-wanted](https://github.com/eclipse-jkube/jkube/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22+) issues instead in order to continue you contributor journey in the project.

**Comment by pujakarakoti07** [comment]
I want to fix this issue can you please assign me?

**Comment by manusa** [comment]
Hi @pujakarakoti07, you already have #3368 assigned to you.

**Comment by bhavana61** [comment]
@rohanKanojia can i have this assigned? I wanna go through the contributions process as I am facing some issue while pushing it

**Comment by rohanKanojia** [comment]
@bhavana61 : Have you forked the project? Please go through my blogpost [How to contribute to Eclipse JKube](https://rohaan.medium.com/how-to-contribute-to-eclipse-jkube-f3bc7f902bf4) for step by step guide in contributing process.

**Comment by bhavana61** [comment]
Yes..I have gone through the process and I have commited the change

**Comment by rohanKanojia** [comment]
@bhavana61 : Unfortunately ECA check is failing, I'm not sure if it happened by accident but you've created the commit with your work email account (not sure if it's the same email with which you signed ECA)

**Comment by bhavana61** [comment]
i am not sure why this happened...I have changed the git config user.name and email before commiting...how can I change this?

**Comment by bhavana61** [comment]
@rohanKanojia I have closed the old PRs with different account commits. Reopened the new one. can you check once?

**Comment by rohanKanojia** [comment]
@bhavana61 : How did you make these changes? If you open the project in IntelliJ and load file there IntelliJ would give you a suggestion to replace the anonymous class with lambda.
